code/modules/mob/living/carbon/human/species/station/slime.dm

```diff
@@ -0,0 +1,91 @@
+
+/datum/species/slime
+	name = "Slime"
+	name_plural = "slimes"
+
+	default_form = FORM_SLIME
+	obligate_form = TRUE
+	reagent_tag = IS_SLIME
+	unarmed_types = list(/datum/unarmed_attack/slime_glomp)
+	flags = NO_SLIP | NO_BREATHE | NO_BLOOD | NO_SCAN | NO_MINOR_CUT
+	siemens_coefficient = 3 //conductive
+	darksight = 3
+	virus_immune = TRUE
+	breath_type = null
+	poison_type = null
+	hunger_factor = DEFAULT_HUNGER_FACTOR * 2
+	spawn_flags = IS_RESTRICTED
+
+	burn_mod = 1
+	brute_mod = 1
+	toxins_mod = -1 // Flips the affects of toxins, so dealing toxins heals well reducing toxins damages - R4D6|Trilby
+	oxy_mod = 0
+
+	has_process = list(
+		BP_BRAIN = /obj/item/organ/internal/brain/slime
+		)
+
+	breath_type = null
+	poison_type = null
+
+	bump_flag = SLIME
+	swap_flags = MONKEY|SLIME|SIMPLE_ANIMAL
+	push_flags = MONKEY|SLIME|SIMPLE_ANIMAL
+
+	has_process = list(
+		OP_HEART   = /obj/item/organ/internal/heart/slime,
+		BP_BRAIN   = /obj/item/organ/internal/brain/slime,
+		OP_EYES    = /obj/item/organ/internal/eyes/slime,
+		OP_STOMACH = /obj/item/organ/internal/stomach/slime
+		)
```

Comment:
First, `has_process` is already defined above.
Second, while I don't know if they need a stomach to eat, they definitively don't need eyes or a heart to see. They are supposed to only need the core, so at worse just make the core/brain do the stomach stuff too.

Reply:
Also they don't have blood, so a heart is useless.